.gitmodules

```diff
@@ -62,3 +62,13 @@
 	path = src/modules/uxrce_dds_client/Micro-XRCE-DDS-Client
 	url = https://github.com/PX4/Micro-XRCE-DDS-Client.git
 	branch = px4
+[submodule "src/modules/cdrstream/cyclonedds"]
+	path = src/lib/cdrstream/cyclonedds
+	url = https://github.com/eclipse-cyclonedds/cyclonedds
```

Comment:
Let's get these forked and carried under the PX4 org. In the past we've had submodules change out from under us in older releases and it was painful.

Reply:
@PetervdPerk-NXP here are the forks (you should have full access).

 - https://github.com/PX4/cyclonedds
 - https://github.com/PX4/rosidl
 - https://github.com/PX4/zenoh-pico

I usually keep the master/main branch in sync with the upstream repo, then create a `px4` branch so it's obvious.